docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -1,5 +1,66 @@
 # 10.2.x Release Notes
 
+## 10.2.1
+
+This release includes:
+
+* Improved source-compatibility with Akka HTTP 10.1.x [#3489](https://github.com/akka/akka-http/pull/3489)
+* Re-introduction of the lingerTimeout option [#3456](https://github.com/akka/akka-http/pull/3456)
```

Comment:
It was more of a reenabling since we never removed the option but it just stopped working :)